### PR TITLE
Remove init from app delegate

### PR DESCRIPTION
**Pull request by StevenDiviney**
# Changes
* Moved sdk init out of app delegate
* Moved init to ViewController, added state to track when SDK is ready and a super helpful alert for to let user know when it's not.

## Reviewers
@tecknut @RonanMcH @oarrabi 

## Risks
*  Low

### CODE REVIEW

File: ZendeskSDK Urban Airship Example App/AppDelegate.m [hunk 1]
```diff
@@ -100,7 +96,7 @@ - (void)application:(UIApplication *)application didRegisterForRemoteNotificatio
 - (void)application:(UIApplication *)application didReceiveRemoteNotification:(NSDictionary *)userInfo fetchCompletionHandler:(void (^)(UIBackgroundFetchResult))completionHandler
 {
     
-    [ZDKPushUtil handlePush:[[NSDictionary alloc] initWithDictionary:pushUtilUserInfo]
+    [ZDKPushUtil handlePush:[[NSDictionary alloc] initWithDictionary:userInfo]
              forApplication:application
```
Comment (RonanMcH) [inline ZendeskSDK Urban Airship Example App/AppDelegate.m]: Did you investigate about the `userInfo` variable and what exactly happened to the `pushUtilUserInfo`?

Comment (StevenDiviney) [inline ZendeskSDK Urban Airship Example App/AppDelegate.m]: nope, good point I'll take a look

Comment (StevenDiviney) [inline ZendeskSDK Urban Airship Example App/AppDelegate.m]: yeah it's fine as it is now, it was:

NSMutableDictionary *pushUtilUserInfo = [[NSMutableDictionary alloc] initWithDictionary:userInfo];
[pushUtilUserInfo setObject:userInfo[@"tid"] forKey:@"zendesk_sdk_request_id"];

But this was basically brought into the SDK.

**Comment by RonanMcH** [comment]
:+1:

**Pull request merged**